shell/platform/darwin/ios/framework/Source/FlutterMetalLayer.mm

```diff
@@ -172,6 +172,18 @@ - (instancetype)init {
   return self;
 }
 
+- (MTLPixelFormat)pixelFormat {
+  return _pixelFormat;
+}
+
+- (void)setPixelFormat:(MTLPixelFormat)pixelFormat {
+  _pixelFormat = pixelFormat;
+
+  // FlutterView updates pixel format on it's layer, but the overlay views
```

Comment:
Not magic, just missing until https://github.com/flutter/engine/pull/48190

Reply:
I just assumed it worked on main :) should have checked.

Reply:
That is very generous of you 😆